.github/FUNDING.yml

```diff
@@ -0,0 +1,12 @@
+# These are supported funding model platforms
+
+github: [kamthamc]
```

Comment:
Heh - we should remove this file. Don't think it makes sense.

Reply:
Sorry, didn't see that. Removed it